<launch>

  <arg name="layout" default="0" doc="Gamepad layout to load. Default is 0 for Logitech F130."/>
  
  <include file="$(find unitree_legged_real)/launch/real.launch">
    <arg name="ctrl_level" value="HIGHLEVEL"/>
  </include>

  <node pkg="go1controller"
        type="joystick_control"
        respawn="true"
        name="joystick_controller"
        output="screen"
        args="$(arg layout)"/>

</launch>